src/shims.h

```diff
@@ -41,6 +41,8 @@
 #if HAVE_PTHREAD_WORKQUEUES
 #if __has_include(<pthread/workqueue_private.h>)
 #include <pthread/workqueue_private.h>
+#elif DISPATCH_USE_INTERNAL_WORKQUEUE
+#include "event/workqueue_internal.h"
```

Comment:
I think since you merged #248, the elif is dead code.  But fine to merge as is to save time; I can double check after the merge and submit a PR to remove this if I'm right.